src/tokens/mainnet.json

```diff
@@ -91,6 +91,16 @@
       "name": "Bread",
       "symbol": "BREAD"
     },
+    {
+      "address": "0xddd3ea5de9c70973e224d938b8f392ec4cc0171c",
+      "decimals": 18,
+      "image": "bull-ish-v2.svg",
+      "isTransferable": false,
+      "name": "BULL ISH V2",
```

Comment:
One more attribute we need is `notPriced` or `isPriced` we need on BE to understand that no need to compute the APR/TVL and price.